content/docs/guides/pipelines/deploy-pipelines-service.md

```diff
@@ -3,195 +3,67 @@ title = "Set Up and Deploy the Pipelines Service"
 description = "Deploy the Kubeflow Pipelines service"
 weight = 3
 toc = true
-bref = "This page guides you through the steps to deploy Kubeflow, including the Kubeflow Pipelines service."
 
 [menu.docs]
   parent = "pipelines"
   weight = 3
 +++
-## Deploying Kubeflow Pipelines on a local Minikube cluster
-
-Install [Docker](https://docs.docker.com/install/), socat 
-(`sudo apt-get install socat` - needed for port forwarding) and 
-[Minikube](https://github.com/kubernetes/minikube#installation) (or 
-[install Minikube without VM](https://github.com/kubernetes/minikube#linux-continuous-integration-without-vm-support)).
-
-Then continue to [deploy Kubeflow Pipelines](#deploy-kubeflow-pipelines)
-as described below.
-
-## Deploying Kubeflow Pipelines on GKE
-
-You can use [Cloud Shell](https://cloud.google.com/shell/docs/quickstart) to run 
-all the commands in this guide. 
-
-Alternatively, if you prefer to install and interact with 
-Kubernetes Engine (GKE) from your local  machine, make sure you have 
-[gcloud CLI](https://cloud.google.com/sdk/) and 
-[kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#download-as-part-of-the-google-cloud-sdk) 
-installed locally.
-
-Follow the 
-[GCP  guide](https://cloud.google.com/resource-manager/docs/creating-managing-projects) 
-to create a GCP project.
-
-Enable the GKE API on 
-[the GCP Console](https://console.developers.google.com/apis/enabled). You can 
-find more details about enabling 
-[billing](https://cloud.google.com/billing/docs/how-to/modify-project?#enable-billing), 
-as well as activating the 
-[GKE API](https://cloud.google.com/kubernetes-engine/docs/quickstart#before-you-begin).
-
-We recommend that you use Cloud Shell from the GCP Console to run the following 
-commands. Cloud Shell starts with an environment already logged in to your 
-account and set to the currently selected project. The following two commands 
-are required only in a workstation shell environment; you don't need to run
-these commands if you're using Cloud Shell:
 
-```bash
-gcloud auth login
-gcloud config set project [your-project-id]
-```
+Starting from [V0.1.3](https://github.com/kubeflow/pipelines/releases/tag/0.1.3), Kubeflow Pipelines becomes one of the Kubeflow core components. It's automatically deployed during Kubeflow deployment. 
+
+Note: Due to 
+[kubeflow/pipelines#345](https://github.com/kubeflow/pipelines/issues/345) and 
+[kubeflow/pipelines#337](https://github.com/kubeflow/pipelines/issues/337), Kubeflow Pipelines depends on GCP services and some of the funcitonalities are not supported by non-GKE cluster.
+For best user experience, please deploy Kubeflow through GKE. Please refer to [GKE setup](/docs/started/getting-started-gke/) for instruction.
```

Comment:
Can we please reword this last sentence to:
You can try it currently with a Kubeflow deployment on GKE. Please refer to [GKE setup](/docs/started/getting-started-gke/) for instructions.

Reply:
Also as an FYI, tech writers avoid the use of the word "please" as it can have an ambiguous tone, especially for readers whose first language is not English. Instead, it's clearer just to tell people what to do:

```
You can try it currently with a Kubeflow deployment on GKE. See the [GKE setup guide](/docs/started/getting-started-gke/).
```

Reply:
done